app/components/timeline_entry/component.rb

```diff
@@ -199,5 +199,21 @@ def stage_changed_vars
           ).strip,
       }
     end
+
+    def application_declined_vars
+      {
+        declined:
+          if timeline_event.application_form.declined_at?
+            render(
+              StatusTag::Component.new(
+                "declined",
+                class_context: "timeline-event",
+              ),
+            ).strip
+          else
+            ""
+          end,
```

Comment:
It doesn't look like this variable is used anywhere so I think we can remove it.